src/project-css/app/Http/Controllers/TableController.php

```diff
@@ -499,37 +533,100 @@ public function getColLst($tblNme) {
   }
 
   /**
-  * Worker employs following algorithm:
+  * Store takes a requested file import and adds it to the job queue for later processing
+  **/
+  public function store(Request $request) {
+    // set messages array to empty
+    $messages = [ ];
+
+    Log::info('File Import has been requested for table ' . $request->tblNme . ' using flat file ' . $request->fltFle);
+    // add job to queue
+    $this->dispatch(new FileImport($request->tblNme, $request->fltFle));
+    $message = [
+      'content'  =>  $request->fltFle.' has been queued for import to ' . $request->tblNme . ' table. It will be available shortly.',
+      'level'    =>  'success',
+    ];
+    array_push($messages, $message);
+    session()->flash('messages', $messages);
+    return redirect()->route('tableIndex');
+  }
+
+  // if 2 lines are read that do not contain enough fields we will attempt to
+  // merge them to get the required number of Fields we assume that the last array
+  // item in $tkns1 is continued in the first item of $tkns2 so they will be
+  // combined.
+  public function mergeLines($tkns1, $tkns2) {
+    $numItem = count($tkns1) - 1;
+    $tkns1[$numItem] = $tkns1[$numItem] . ' ' . $tkns2[0];
+    unset($tkns2[0]);
+    return( (count($tkns2) > 0) ? array_merge($tkns1, $tkns2) : $tkns1 );
+    // if (count($tkns2) > 0) {
```

Comment:
remove commented code  thats not being used.

Reply:
done